### DISCUSSION TITLE
Brace expansion equivalent

### DISCUSSION THREAD

**Discussion by rosshadden** [nushell/nushell#4259] [Q&A] [answered]
One of my favorite lesser-known features of bash and zsh is brace expansion.

```shell
# renames some.file => some.file.bak
mv some.file{,.bak}
```

Is there an equivalently convenient way to do this in nushell?

**Comment by rosshadden** [comment]
Ah I just found https://github.com/nushell/nushell/issues/2317 while searching discord. Not sure how I didn't see it when I searched github for the very same query :sweat_smile:.

**Comment by id-ekaagr** [comment, marked as answer]
You need this in nu: ("{1..2}" | str expand), it's bash eq is {1..2}. Thank me later.
